runtime/vm/exceptiondescribe.c

```diff
@@ -330,7 +330,14 @@ iterateStackTrace(J9VMThread * vmThread, j9object_t* exception, callback_func_t
 					pruneConstructors = FALSE;
 #endif
 					romClass = findROMClassFromPC(vmThread, methodPC, &classLoader);
-					if (NULL != romClass) {
+					if ((NULL != romClass)
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+						&& !J9ROMCLASS_IS_PRIMITIVE_VALUE_TYPE(romClass)
```

Comment:
could we peek but then always use the qtype if one exists?

Reply:
I believe it depends on whether Ltype can be instanciated, whether we can run methods on Ltype.